packages/shared/src/broker.ts

```diff
@@ -152,5 +170,8 @@ export async function requestSwapDepositAddress(
       cfParameters: undefined,
     },
     boostFeeBps,
+    opts.affiliateBrokers?.length
+      ? opts.affiliateBrokers.map(({ account, commissionBps }) => ({ account, bps: commissionBps }))
+      : undefined,
```

Comment:
if you don't accept the other suggestion
```suggestion
    opts.affiliateBrokers?.map(({ account, commissionBps }) => ({ account, bps: commissionBps })),
```